newtemplates/weatherapp2.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
 
-    <title>Weather App</title>
+    <title>Weather App Page 2</title>
     <link rel="stylesheet" href="weatherapp2.css">
 </head>
 <body>
```

Comment:
Why is London,uk hard coded, add variable references, variables are stored in cars class, refer run.py for variables